build.sh

```diff
@@ -62,6 +62,10 @@ mkdir -p $OUTDIR/lib/bootstrap
 cp node_modules/bootstrap/dist/js/bootstrap.min.js $OUTDIR/lib/bootstrap/bootstrap.min.js
 cp node_modules/bootstrap/dist/css/bootstrap.min.css $OUTDIR/lib/bootstrap/bootstrap.min.css
 
+# to update node-forge library, which is missing the non-minified version in dist, we have to build it manually
```

Comment:
Once you manage to get a working mechanism to build forge, please update the lines here so that we can re-build it easily next time.

Reply:
Even if it was just by hacking around the build mechanism of forge, please document it here or somewhere so that we can repeat it if we need to, later

Reply:
Yeah, I've got that but I'll try to get the build mechanism upstreamed in node-forge as more people may be interested in this version. It's also a regression as 0.9.1 worked with non-eval, non-minimized fashion but 0.10 doesn't.

If they're not interested in the build PR I'll add more thorough instructions.

Reply:
Ok. Then I'll merge as is